microprofile-jwt/src/main/java/org/jboss/eap/qe/microprofile/jwt/tools/LogListener.java

```diff
@@ -0,0 +1,62 @@
+package org.jboss.eap.qe.microprofile.jwt.tools;
+
+import lombok.extern.slf4j.Slf4j;
+import org.apache.commons.io.input.Tailer;
+import org.apache.commons.io.input.TailerListener;
+
+import java.util.concurrent.atomic.AtomicBoolean;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+/**
+ * Basic tailer log listener, base on waiting on pattern to appear.
+ */
+@Slf4j
+public final class LogListener implements TailerListener {
```

Comment:
Can we have different approach than depending on log ?

Log can get big ...

Reply:
Do you suggest some other approach? See my reasoning: https://github.com/jboss-eap-qe/eap-microprofile-test-suite/issues/39#issuecomment-558021120